ads/google/imaVideo.js

```diff
@@ -1153,20 +1157,22 @@ export function playVideo() {
  * @visibleForTesting
  */
 export function pauseVideo(event = null) {
-  videoPlayer.pause();
-  playerState = PlayerStates.PAUSED;
-  // Show controls and keep them there because we're paused.
-  clearInterval(hideControlsTimeout);
-  if (!adsActive) {
+  if (adsActive) {
```

Comment:
the check for adsActive has been moved from onMessage event https://github.com/ampproject/amphtml/pull/23660/files#diff-64400de024de2e5299df7084028aabd9L1437